pkg/webhook/parser.go

```diff
@@ -448,6 +449,8 @@ func (g Generator) Generate(ctx *genall.GenerationContext) error {
 	var mutatingWebhookCfgs admissionregv1.MutatingWebhookConfiguration
 	var validatingWebhookCfgs admissionregv1.ValidatingWebhookConfiguration
 
+	groupRegex := regexp.MustCompile(`^[a-z][-a-z0-9.]*[a-z0-9]$`)
```

Comment:
maybe worth to have some positive and negative unit tests for this to ensure we (don't) error out on the right ones.